packages/docs/src/styles/pages/layout.scss

```diff
@@ -21,33 +21,38 @@ body {
   flex: 1 0 auto;
 }
 
+.page-content__content {
+  > h2 {
+    @extend .ds-h2;
```

Comment:
I was under the impression we wanted to avoid `@extend` rules?

Reply:
Well the other option would be to use a plugin with gatsby to add the heading classes to our markup content, what do you think @pwolfert ?

Reply:
I actually looked into this while working on these and it was something of a pain to get going.. I agree using extend isn't great but the other option may need another ticket maybe and could be expanded to setup classes for all our elements to utilize our own styling classes.

Reply:
It's not in code that we're releasing to other teams, so I think it's a fine solution for now.